plugins/aimirim.fordiac.ide.deployment.uao/plugin.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+<plugin>
+   <extension
+         point="org.eclipse.fordiac.ide.deployment.devicemanagementinteractor">
+      <DeviceManagementInteractorProvider
+            class="aimirim.fordiac.ide.deployment.uao.providers.UAODevMgmtInteractorProvider">
+      </DeviceManagementInteractorProvider>
+   </extension>
+   <extension
```

Comment:
I don't think you need this menu contribution here.

Reply:
That one came along when I've duplicated the opcua deploy as a baseline.
Removed in 749c7f5.